diagnostic.md

```diff
@@ -40,14 +67,15 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a representation of the HTML.  It is made of nodes and can be used by jQuery to traverse through the browser's HTML code.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.